src/FluentCheck.ts

```diff
@@ -13,11 +13,20 @@ class FluentResult {
   }
 }
 
-export type FluentConfig = { sampleSize?: number, shrinkSize?: number }
+export type FluentConfig = { sampleSize?: number, shrinkSize?: number}
 
 export class FluentCheck<Rec extends ParentRec, ParentRec extends {}> {
-  constructor(protected readonly parent: FluentCheck<ParentRec, any> | undefined = undefined,
+  public prng: {
```

Comment:
You should define a type (using a TS type alias) instead of repeating this over and over!